### PR TITLE
Docs site changes

### PR DESCRIPTION
**Pull request by KrooshalUX**
### Description
Updates the getting started markdown file to remove references no longer relevant, including sample projects. 
 
### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [X] All tests pass
  - [X] `yarn lint`
  - [X] `yarn test-unit`
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: src-docs/src/views/guidelines/getting_started.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ yarn add @opensearch-project/oui @elastic/datemath moment prop-types
 
 We depend upon the version of node defined in [.nvmrc](.nvmrc).
 
-You will probably want to install a node version manager. [nvm](https://github.com/creationix/nvm) is recommended.
+We recommend installing a node version manager: [nvm](https://github.com/creationix/nvm) is recommended.
```
Comment (BSFishy) [inline src-docs/src/views/guidelines/getting_started.md:22]: Nitpick: This link redirects to https://github.com/nvm-sh/nvm, so I think we could just have that directly.

**Review by BSFishy — COMMENTED** [review]
It looks like you included project files (`OUI/OUI.iml` and `oui.iml`), which I don't think should be included. Other than that, lgtm 👍

**Review by AMoo-Miki — CHANGES_REQUESTED** [review]
Can you please add *.iml to the `gitignore`?

**Comment by KrooshalUX** [comment]
@BSFishy changes made
@AMoo-Miki added *.iml to gitignore - though I dont see that commit showing up here if it should.

also I made some manual commits because I was having an issue in the IDE (now resolved) , and I forgot to sign them off. I tried to follow various how-to's on resolving, but nothing seemed to work other than manually setting it to pass. The option to do so only showed on one commit, however. Sadly, I think I need some hand holding to resolve.

**Review by bandinib-amzn — APPROVED** [review]
LGTM!

**Review by BSFishy — APPROVED** [review]

**Review by joshuarrrr — APPROVED** [review]

**Review by AMoo-Miki — APPROVED** [review]

**Pull request merged**